Dockerfile.in

```diff
@@ -1,6 +1,10 @@
-FROM ARG_FROM
+ARG base_image=registry.access.redhat.com/ubi8/ubi-minimal:8.1
```

Comment:
do we use this file anymore? Let's remove it if not

Reply:
not sure... let me check. 
make file is using package.sh

Reply:
it's part of goreleaser :( 
nice catch

Reply:
goreleaser uses https://github.com/kanisterio/kanister/blob/master/.goreleaser.yml#L43

Reply:
we probably need a way of auto get docker files for goreleaser

Reply:
we do use `Dockerfile.in` in k10:
```
 636 docker_package_kanister() {
 637     local image_tag=${1:?"Image tag unspecified. Pick a tag that is unique for every code change."}
 638     pushd "${KANISTER_PATH}"
 639     cp ${BASE_DIR}/go/bin/controller controller
 640     GO111MODULE=on BIN=controller ARCH=amd64 BASEIMAGE=alpine \
 641         IMAGE=${DOCKER_ECR}/kanister VERSION=${image_tag}     \
 642         SOURCE_BIN=controller                                 \
 643         ./build/package.sh
 644     popd
```